composer.json

```diff
@@ -14,10 +14,10 @@
         "doctrine/orm": "~2.0",
         "nelmio/alice": "~1.5||~2.0",
         "php": ">=5.4",
-        "symfony/dependency-injection": "~2.0",
-        "symfony/finder": "~2.0",
-        "symfony/framework-bundle": "~2.0",
-        "symfony/http-kernel": "~2.0"
+        "symfony/dependency-injection": "~2.0|~3.0",
```

Comment:
Please, update the constraint to `~2.3|~3.0`. Since Symfony < 2.3 is not supported anymore, we can easily drop the support.